.eslintrc.json

```diff
@@ -68,7 +69,23 @@
         "react/no-string-refs": 2,
         "react/prefer-stateless-function": 2,
         "react/self-closing-comp": 2,
-        "react/sort-comp": 2
+        "react/sort-comp": 2,
+        "@angular-eslint/component-class-suffix": 2,
```

Comment:
Can these be moved to the top of the list (alphabetical order)?

Reply:
Makes sense. Moved.